externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkOperation.scala

```diff
@@ -123,7 +123,7 @@ abstract class SparkOperation(session: Session)
 
   protected def eventEnabled: Boolean = true
 
-  if (eventEnabled) EventBus.post(SparkOperationEvent(this))
+  if (handle != null && eventEnabled) EventBus.post(SparkOperationEvent(this))
```

Comment:
in which case that `handle == null` is true?

Reply:
For execute statement， it override the handle.

Reply:
Otherwise，npe thrown.

Reply:
fixed